marklogic-data-hub-central/ui/e2e/setup.sh

```diff
@@ -17,7 +17,7 @@ cd hc-qa-project
 if $DHS
 then
         env=dhs
-        sed -i '' "s/mlHost=/mlHost=$mlHost/g" gradle-dhs.properties
+        sed -i  "s/mlHost=/mlHost=$mlHost/g" gradle-dhs.properties
```

Comment:
we need '' for mac. Is it not working on linux/windows?

Reply:
No its not working on linux. We will be running on linux machines

Reply:
Can we try using `perl -i -pe"s/mlHost=/mlHost=$mlHost/g" gradle-dhs.properties` instead.
Works in mac and hopefully works on all platforms.